services/upload/src/main/resources/application.yaml

```diff
@@ -25,7 +25,7 @@ services:
     certificate: ${VAULT_EFGS_BATCHIGNING_CERTIFICATE}
     privatekey-password: ${VAULT_EFGS_BATCHSIGNING_PASSWORD}
     test-data:
-      keys: 750
+      keys: 6000
```

Comment:
shouldn't we have 0 in production code and a higher number then in ` services/upload/src/test/resources/application.yaml` ?

Reply:
This property will only be picked up if we have the `testdata` profile active. Now thinking about it we should move this property to a new file `application-testdata.yaml` like we have in distribution. 

As for the amount: 6000 keys are the per day generation on Upload. On distribution the test data is 1000 per hour. So I suppose we could change to 24k keys on the upload to match the same volume of data on distribution?